{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -190,7 +190,6 @@
 ANYMAIL = {
     "SENDGRID_API_KEY": env("SENDGRID_API_KEY"),
     "SENDGRID_GENERATE_MESSAGE_ID": env("SENDGRID_GENERATE_MESSAGE_ID"),
```

Comment:
As per my suggestion in the issue, I don't think we need `SENDGRID_GENERATE_MESSAGE_ID` either:

```suggestion
```

Reply:
Sure @browniebroke ,

- I have removed env SENDGRID_GENERATE_MESSAGE_ID from  from \cookiecutter-django{{cookiecutter.project_slug}}\config\settings\production.py as well. Please review the changes.
- I ran tox command to run the default tests and they completed successfully.